src/main.js

```diff
@@ -82,7 +85,7 @@ function createWindow() {
     width: 600,
     height: 470,
     // frame: false,
-    icon: path.join(__dirname, '../../assets/icon.ico'),
+    icon: isDebug ? path.join(__dirname, '../../assets/icon.ico') : undefined,
```

Comment:
```suggestion
    icon: isDebug ? path.join(app.getAppPath(), 'assets/icon.ico') : undefined,
```
just checking if we can use this here too?

Reply:
I checked, it'is okay, I made the change, thanks for the comment.